gradle/libs.versions.toml

```diff
@@ -13,7 +13,7 @@ apache-commons-compress = '1.26.0'
 
 assertj = '3.25.3'
 avro = '1.11.4'
-byte-buddy = '1.14.19'
+byte-buddy = '1.18.1'
```

Comment:
The first version of `byte-buddy` that supports Java 25 is [1.17.5](https://github.com/raphw/byte-buddy/releases/tag/byte-buddy-1.17.5)